.travis.yml

```diff
@@ -5,6 +5,7 @@ bundler_args: --without debugging
 script:
 - bundle exec rake
 - bundle exec ataru check
+- bundle exec mutant --include lib --require reek --use rspec --since master "Reek*"
```

Comment:
You probably want to limit concurrency on CI. Mutants default is to spawn as many jobs as processors seen in `/proc/cpuinfo`, on CI environments like Travis this does not reflect the containers resource allocation.

I suggest adding `--jobs 4` for this project, 4 jobs had been proven a reasonable value in environments like Travis.